src/Resources/config/routing/api/gallery.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<routes xmlns="http://symfony.com/schema/routing" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/routing https://symfony.com/schema/routing/routing-1.0.xsd">
+    <route id="get_galleries" path="/galleries.{_format}" methods="GET" controller="Sonata\MediaBundle\Controller\Api\GalleryController::getGalleriesAction" format="json">
+        <requirement key="_format">json|xml|html</requirement>
+    </route>
+    <route id="get_gallery" path="/galleries/{id}.{_format}" methods="GET" controller="Sonata\MediaBundle\Controller\Api\GalleryController::getGalleryAction" format="json">
+        <requirement key="_format">json|xml|html</requirement>
+    </route>
+    <route id="get_gallery_medias" path="/galleries/{id}/medias.{_format}" methods="GET" controller="Sonata\MediaBundle\Controller\Api\GalleryController::getGalleryMediasAction" format="json">
+        <requirement key="_format">json|xml|html</requirement>
+    </route>
+    <route id="get_gallery_galleryhasmedias" path="/galleries/{id}/galleryhasmedias.{_format}" methods="GET" controller="Sonata\MediaBundle\Controller\Api\GalleryController::getGalleryGalleryhasmediasAction" format="json">
```

Comment:
I'd like to have this list sorted by id, but it's not really a blocker.

Reply:
I check it. Better keep it in this way becouse order in config is equal to order in nelmio doc. Its look much better when we keep this order.